README.md

```diff
@@ -19,6 +19,8 @@ gems:
   - jekyll-feed
 ```
 
+:bulb: If you are using Jekyll `^3.5.0`, you probably want to use the `plugins` key instead of `gems`.
```

Comment:
I'm not sure everybody understand this notation. What about plain text?

If you are using Jekyll version `3.5.0` or greater…